app/src/pages/api/business/resolvers/queries/getBusinessCampaigns.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryResolvers } from "api/codegen/resolvers-types";
+import { ResolversContext } from "api/graphql";
+import { getPageContentByBusinessCampaignSlug } from "lib/wordpress/getPageContentBySlug";
+
+const getBusinessCampaigns: QueryResolvers["getBusinessCampaigns"] = async (_, { database }: ResolversContext) => {
+  try {
+    const data = await database.business.businessFundingCampaignPlanModel.findAll();
+
+    if (!data || !data.length) {
+      throw new Error(`getBusinessCampaigns: unable to find business campaigns records: ${JSON.stringify(database)}`);
+    }
+
+    const businessCampaignsData = data.map(async (bc) => {
```

Comment:
Let's call it as an action: 

```suggestion
    const mergeCampaignDataWithWordpressContent = data.map(async (bc) => {
```